crates/sui-cost/src/cost.rs

```diff
@@ -89,7 +90,7 @@ pub enum SuiNativeCostIndex {
 
 // Native costs are currently flat
 // TODO recalibrate wrt bytecode costs
-pub fn native_cost_schedule() -> Vec<GasCost> {
+pub fn _native_cost_schedule() -> Vec<GasCost> {
```

Comment:
Why the name change? Are we no longer using this function (in which case it should probably be removed)?

Reply:
We're not using it but we will use the constants defined there in a different way. We need to port them over carefully.